tools/clang/test/HLSLFileCheckLit/hlsl/types/matrix/constMat.hlsl

```diff
@@ -1,4 +1,5 @@
 // RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
+// RUN: %dxc -E main -T ps_6_0 -Zpr %s | FileCheck %s
 
 // CHECK: fmul
 // CHECK: 2.000000e+00
```

Comment:
The quality of the FileCheck `CHECK` directives in this test is a bit suspect. If we're moving these around should we also improve the tests?

Reply:
Updated.